CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 * Properly consider `b > c` to be a superselector of `a > b > c`, and similarly
   for other combinators.
 
+* Allow loading the same module more than once from the same module
+  configuration.
```

Comment:
This is a bit confusing because users might expect "the same configuration" to mean "structurally equivalent" rather than the hierarchical-identity scheme we use here. From a user's perspective, this is more like "Don't throw an error when the same module is `@forward`ed multiple times through a configured module".